ledger-tool/src/main.rs

```diff
@@ -1,6 +1,6 @@
 use clap::{crate_version, App, Arg, SubCommand};
 use solana::bank::Bank;
-use solana::ledger::{read_ledger, verify_ledger};
+use solana::ledger::{read_ledger, tail_ledger, verify_ledger};
```

Comment:
ack, so I'm working on a [patch](https://github.com/solana-labs/solana/pull/2298/files#diff-3125b196bc2e46166465885cd9ad66ad) that'll conflict hard with this -- I'm porting ledger-tool from src/ledger to src/db_ledger and db_ledger won't permit another process (ledger-tool) to open the ledger at the same time as the fullnode has db_ledger open (*I think*).

Reply:
@mvines / @garious what's the best alternative for real-time streaming a ledger, would it be replicator API? (haven't dug into that part of the code... yet)